src/gui/MainWindow.cpp

```diff
@@ -576,17 +549,6 @@ bool MainWindow::deleteEventCallback(GtkWidget* widget, GdkEvent* event, Control
 	return true;
 }
 
-bool MainWindow::configureEventCallback(GtkWidget* widget, GdkEventConfigure* event, Control* control)
```

Comment:
Why is this removed, was this dead code from GTK2? ("configure-event")

Reply:
I added this some commits before but it didn't catched when the application was resized using tiling. I moved to the signal size-allocate from the widget and the ZoomControl is listening to it. Now everything is working.